explainer.md

```diff
@@ -1,624 +1,397 @@
-      isolation: isolate;
-      mix-blend-mode: [plus-lighter](https://drafts.fxtf.org/compositing/#porterduffcompositingoperators_plus_lighter);
-    }
-
-    ::shared-old(header-id), ::shared-new(header-id) {
-      position: absolute;
-      width: 100%;
-      height: 100%;
-    }
-
-    ::shared-old-root {
-      /* This is the saved output referenced in step 3. */
-      content: cached-element(html);
-    }
-    ::shared-old(header-id) {
-      content: cached-element(header-id);
-    }
-
-    ::shared-new-root {
-      content: element(html);
-    }
-    ::shared-new(header-id) {
-      content: element(header-id);
-    }
-
-    /* Default animations added by the UA which can be overridden by the developer stylesheet/script. */
-    @keyframes ::shared-new-fade-in {
-      0% {opacity: 0;}
-      100% {opacity: 1;}
-    }
-    ::shared-new-root, ::shared-new(header-id) {
-      animation: ::shared-new-fade-in 0.25s;
-    }
-
-    @keyframes ::shared-old-fade-out {
-      0% {opacity: 1;}
-      100% {opacity: 0;}
-    }
-    ::shared-old-root, ::shared-old(header-id) {
-      animation: ::shared-old-fade-out 0.25s;
-    }
-
-    /* Generated for each shared element with the syntax shared-container-sharedid. */
-    @keyframes ::shared-container-header-id {
-      from {
-        width: 100px;
-        height: 100px;
-        transform: translate(8px, 308px);
-      }
-    }
-    ::shared-container(header-id) {
-      animation: ::shared-container-header-id 0.25s;
-    }
-    ```
-
-8.  When the transition finishes, remove all pseudo elements from the top layer.
-    The transition finishes when there is no active animation on any pseudo
-    element. See
-    [issue 64](https://github.com/WICG/shared-element-transitions/issues/64) for
-    discussion on this.
-    
-An example simulating the steps above using the existing `element()` function
-is [here](https://jsbin.com/niqiqididu/edit?html,output) (open in Firefox[^2]).
-
-[![Video Link for Shared Element Transition](https://img.youtube.com/vi/QzGEBUW-3U8/0.jpg)](https://youtu.be/QzGEBUW-3U8)
-
-## Animating Box Decoration CSS Properties
-
-A common capability desirable during transitions is to interpolate styles like
-border-radius that change the element's shape. The
-[container transform](https://material.io/design/motion/the-motion-system.html#container-transform)
-examples show a visual demo of that. Painting properties like the element's
-border within its image when using the element() function makes this difficult.
-
-Consider the same example as above with the addition of box decorations to
-the shared element.
-
-### Old Document
+- **transition part container**: If the part is created as a "container + child texture", this element will have a width and height of the content area of the original element, and have its computed styles reapplied. If the part is created as a "single texture", this element will have a width and height of the border box of the original element.
```

Comment:
We can mention that the following CSS comes from the UA before the code block? It should be okay since we haven't gotten to any developer code in the explainer up till this point.

The current way felt like reading code in prose but I'm okay either way.